### PR TITLE
Assign GIS Role

### PR DESCRIPTION
**Pull request by james-hollinger**
GISUSER needs to be assigned to the user in the MOH Keycloak when they complete a GIS enrollment.

This new MOH keycloak client will be merged into the existing Keycloak administration client in a future PR.

### CODE REVIEW

File: prime-dotnet-webapi/HttpClients/Keycloak Administration/KeycloakApiDefinitions.cs [hunk 1]
```diff
@@ -13,7 +34,7 @@ public class Role
     }
 
     /// <summary>
-    /// This is not the entire Keyclcoak User Representation! See https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_userrepresentation.
+    /// This is not the entire Keycloak User Representation! See https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_userrepresentation.
```
Comment (AMiKaH) [inline prime-dotnet-webapi/HttpClients/Keycloak Administration/KeycloakApiDefinitions.cs:37]: Nice catch!

**Review by johnnyl6 — APPROVED** [review]
LGTM

**Review by AMiKaH — APPROVED** [review]

**Review by mtpultz — APPROVED** [review]
image

**Review by Paahn — APPROVED** [review]
🕺

**Pull request merged**